README.md

```diff
@@ -44,6 +44,7 @@ You can also deploy multiple files at once:
 
 ```
 @ZeebeDeployment(classPathResources ={"demoProcess.bpmn", "demoProcess2.bpmn"})
+@ZeebeDeployment(bpmnLocations = "classpath*:/bpmn/**/*.bpmn")
```

Comment:
"resources" instead of "bpmnLocations"

Reply:
I have renamed "bpmnLocations" to "resources"